imapbackup38.py

```diff
@@ -643,13 +646,17 @@ def main():
         config = get_config()
         server = connect_and_login(config)
         names = get_names(server,config['thunderbird'],config['nospinner'])
+        exclude_folders = []
         if config.get('folders'):
+            # Takes precedence over --exclude-folders
             dirs = list(map(lambda x: x.strip(), config.get('folders').split(',')))
             if config['thunderbird']:
                 dirs = [i.replace("Inbox", "INBOX", 1) if i.startswith("Inbox") else i
                         for i in dirs]
             names = list(filter(lambda x: x[0] in dirs, names))
-
+        elif config.get('exclude-folders'):
+            # Use --exclude-folders option only when --folders is not set
```

Comment:
This must not be silent - if both appear on the command line, it must fail immediately and not silently do any internal precedence handling.

Reply:
Good call, thanks. I'll make it fail instead.

Reply:
Fixed by c7a53d6